pom.xml

```diff
@@ -923,14 +923,14 @@
                       <!-- antlr4gwt and antlr4-c3-gwt (javax.json) rely on duplicated resources to provide a GWT-compatible
                            implementation for ANTLR -->
                       <dependency>
-                        <groupId>org.antlr</groupId>
+                        <groupId>org.antlr4gwt</groupId>
                         <artifactId>antlr4gwt-runtime</artifactId>
                         <ignoreClasses>
                           <ignoreClass>org.antlr.v4.*</ignoreClass>
                         </ignoreClasses>
                       </dependency>
                       <dependency>
-                        <groupId>org.antlr</groupId>
+                        <groupId>org.antlr4gwt</groupId>
```

Comment:
@Rikkola please validate this which was allegedly missing ☝️

Reply:
This was set so it differs from the actual antrl group id, but it is now the correct one for 4.10.

Reply:
Yes, I mean, this was not part of https://github.com/tarilabs/droolsjbpm-build-bootstrap/pull/2 but I believe it was needed too. Thanks

Reply:
Yep it is needed. Thank you for picking it up.